pyproject.toml

```diff
@@ -50,6 +50,7 @@ dependencies    = [
     "ifaddr==0.2.0",
     "Jinja2==3.1.4",
     "lru-dict==1.3.0",
+    "mashumaro==3.13.1",
```

Comment:
Needed to add mashumaro here otherwise we will have the same issue as with the supervisor lib